cpg-core/src/main/java/de/fraunhofer/aisec/cpg/TranslationConfiguration.java

```diff
@@ -557,25 +559,47 @@ public TranslationConfiguration build() {
 
   @Override
   public String toString() {
-    return new ToStringBuilder(this, ToStringStyle.JSON_STYLE)
```

Comment:
I would not do this in `toString`. ToString should be fast and "lean" and definitly not invoke an extra object mapper, etc. I would propose to extract this into a separate function or invoke the jackson functionality in the Benchmark class

Reply:
Apparently we do not even need our own `toString` function.
Or do we need it somewhere else?

Reply:
Please keep it :) It is used automatically in a lot of places, for example when using the debugger, etc.

Reply:
it was also implicitly called by `printBenchmark` to print out the translation config in a more readable format inside the markdown table.

Reply:
Sorry, I'm not so familiar to the java world.
If I understood you correctly, it's not recommended to use an object mapper for serialization in the context of the `toString` method.
What would you then suggest to use as `toString` for the `TranslationConfiguration`?

Reply:
The suggested `toString` would be the one that was already implemented :)

Reply:
I'm trying to reuse the `getConfigAsJson` inside `toString` to reduce code redundancy.
I tired:
```java
return new ToStringBuilder(this, ToStringStyle.JSON_STYLE).append(getConfigAsJson()).toString();
```
But this fails. 
The following works, but adds the config inside a key:
```java
return new ToStringBuilder(this, ToStringStyle.JSON_STYLE).append("conf", getConfigAsJson()).toString();
```

Can I append a whole Map to the builder? 
Or do I need to iterate all elements and add them separately?

Reply:
I think, I finally got it :)

```java
public String toString() {
    return ToStringBuilder.reflectionToString(this, ToStringStyle.JSON_STYLE);
}
```

Is that what you had in mind?